pkg/api/types.go

```diff
@@ -2,8 +2,20 @@ package api
 
 import docker "github.com/fsouza/go-dockerclient"
 
+const (
+	DefaultNamespace    = "io.openshift.s2i."
+	KubernetesNamespace = "k8s.io/"
```

Comment:
". Kube upstream uses docker suggested labels in the Kubelet (io.foo.bar)"

so yes, I guess so.